ipywidgets/widgets/widget.py

```diff
@@ -247,6 +247,8 @@ def get_view_spec(self):
         help="Name of the view object.").tag(sync=True)
     _view_module_version = Unicode('*',
         help="A semver requirement for the view module version.").tag(sync=True)
+    _view_count = Int(read_only=True,
+        help="The number of views accociated to the model at the front-end").tag(sync=True)
```

Comment:
"associated". Also, can we indicate this is experimental for now? Maybe "EXPERIMENTAL: The number of views of the model displayed in the frontend. This attribute is experimental and may change or be removed in the future."